tools/infrasys/src/shared.rs

```diff
@@ -78,7 +79,10 @@ pub async fn get_stack_outputs(
                 resource_name: stack_name,
             })?[0]
```

Comment:
Is `[0]` safe?

Reply:
The description of the method makes some claims that it would be, but, there's no reason not to handle the case. Fixing.